root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -431,6 +431,17 @@ const testData = [
        input_relationship_type: 'amazon',
        only_valid_entity_types: [],
   },
+  {
+                     input_url: 'https://www.amazon.com/vdp/08c6c18fc7bb4822a166db4834e123f1?ref=dp_vse_rvc_0',
+             input_entity_type: 'release',
+       input_relationship_type: 'amazon',
+    expected_relationship_type: undefined,
+       only_valid_entity_types: [],
+                expected_error: {
+                                  error: 'link to a user video',
+                                  target: 'url',
+                                },
+  },
```

Comment:
I guess Amazon tests were missing `expected_clean_url` because it was returning an empty string in some case. It can now be tested that unwanted URLs are untouched:

```suggestion
  {
                     input_url: 'https://www.amazon.com/vdp/08c6c18fc7bb4822a166db4834e123f1?ref=dp_vse_rvc_0',
             input_entity_type: 'release',
       input_relationship_type: 'amazon',
    expected_relationship_type: undefined,
            expected_clean_url: 'https://www.amazon.com/vdp/08c6c18fc7bb4822a166db4834e123f1?ref=dp_vse_rvc_0',
       only_valid_entity_types: [],
                expected_error: {
                                  error: 'link to a user video',
                                  target: 'url',
                                },
  },
```